activateEvents.js

```diff
@@ -0,0 +1,42 @@
+var CarLot 			= (function (events) {
```

Comment:
Something wacky going on with the spacing before the ```=```, definitely dig the naming of the argument, makes it clear what you're doing internally to this IIFE